src/components/Modal/EditSubkeywordModal.jsx

```diff
@@ -0,0 +1,191 @@
+import { useState } from "react";
+import { useParams } from "react-router-dom";
+
+import asyncEditSubkeyword from "../../api/keyword/asyncEditSubkeyword";
+import { ERROR_MESSAGE, MODAL_TYPE } from "../../config/constants";
+import useBoundStore from "../../store/client/useBoundStore";
+import CreateKeywordButton from "../Button/CreateKeywordButton";
+import KeywordChip from "../Chip/KeywordChip";
+import Portal from "../Common/Portal";
+import PlusIcon from "../Icon/PlusIcon";
+import Label from "../UI/Label";
+import ModalBackground from "./ModalBackground";
+import ModalFrame from "./ModalFrame";
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+
+const EditSubkeywordModal = () => {
+  const modalNode = document.getElementById("modal");
+  const userInfo = useBoundStore((state) => state.userInfo);
+  const { keywordId } = useParams();
+  const keywordName = "임시 대표 키워드 이름";
+  const [inputValue, setInputValue] = useState({
+    includedKeyword: "",
+    excludedKeyword: "",
+  });
+  const [errorMessage, setErrorMessage] = useState({
+    includedKeyword: "",
+    excludedKeyword: "",
+  });
+  const [subkeywordList, setSubkeywordList] = useState({
+    includedKeyword: ["추가 키워드1", "추가 키워드2"],
+    excludedKeyword: ["제외 키워드1", "제외 키워드2"],
+  });
+  const addModal = useBoundStore((state) => state.addModal);
+  const closeModal = useBoundStore((state) => state.closeModal);
+  const queryClient = useQueryClient();
+
+  const editSubkeywordMutation = useMutation({
+    mutationFn: (subkeywordInfo, keywordId) => asyncEditSubkeyword(subkeywordInfo, keywordId),
+  });
+  const handleSubkeywordInputChange = (e, subkeywordType) => {
+    setInputValue((prev) => ({ ...prev, [subkeywordType]: e.target.value }));
+  };
+  const handleCreateSubeywordButtonClick = (subkeywordType) => {
+    if (subkeywordList[subkeywordType].includes(inputValue[subkeywordType])) {
+      return setErrorMessage((prev) => ({
+        ...prev,
+        [subkeywordType]: ERROR_MESSAGE.KEYWORD_DUPLICATED_INPUT_VALUE,
+      }));
+    }
+
+    setSubkeywordList((prev) => ({
+      ...prev,
+      [subkeywordType]: [...prev[subkeywordType], inputValue[subkeywordType]],
+    }));
+    setInputValue((prev) => ({ ...prev, [subkeywordType]: "" }));
+    return setErrorMessage((prev) => ({ ...prev, [subkeywordType]: "" }));
+  };
+  const removeSubkeyword = (subkeywordType, subkeywordNameForRemove) => {
```

Comment:
```suggestion
  const handleSubkeywordRemoveIconClick = (subkeywordType, subkeywordNameForRemove) => {
```

이벤트 핸들러의 네이밍 컨벤션을 따라서, 이름이 좀 길지만 위와 같이 바꿔봐도 좋을 것 같습니다 😄

![image](https://github.com/user-attachments/assets/e4745140-01d4-45f8-b6b2-6bd1f19df9a9)
[React 공식문서: Passing event handlers as props](https://react.dev/learn/responding-to-events#passing-event-handlers-as-props)

Reply:
좋은 아이디어 감사해요, 수정해둘게요! `handle...` 핸들러 함수명 컨벤션을 따르되, 의미가 드러나서 좋습니다.

Reply:
함수명은 `handleRemoveSubKeywordChipClick` 어떨까요? 특정 서브 키워드가 담긴 chip을 삭제한다는 의미를 담으려고 합니다.